crates/bevy_ui/src/ui_node.rs

```diff
@@ -410,3 +410,24 @@ pub struct CalculatedClip {
     /// The rect of the clip
     pub clip: Rect,
 }
+
+/// Indicates that this node has special requirements for the order of depth in which
```

Comment:
```suggestion
/// This [`Node`] entity's front-to-back ordering is not controlled (solely) by its position in the UI hierarchy.
```

Reply:
(remove line below as well)

Reply:
nit: -position+location? (to avoid confusion with `Transform::position`)